app/src/main/java/com/nighthawkapps/wallet/android/ui/home/HomeFragment.kt

```diff
@@ -122,12 +122,14 @@ class HomeFragment : BaseFragment<FragmentHomeBinding>() {
             twig("uiModel exists!")
             onModelUpdated(null, uiModel)
         }
+
+        initTransactionUI()
```

Comment:
This is probably fine but we may need to take a step now to refresh the contents of the adapter. So I would test and see if it's possible for the data to remain stale after returning from the background or from viewing another fragment because setting the adapter each time is another way to refresh the contents and we're potentially losing that refresh with this change.

However, since we start collecting from `walletViewModel.transactions.` in onResume, that might be just as good as a refresh. So, as I mentioned, this is probably fine but worth testing to see if you can break the refresh behavior.